src/microsoft-trydotnet-editor/src/index.css

```diff
@@ -1,10 +1,13 @@
-.body {
+body {
 	width: auto;
 	height: auto;
-	border: 1px solid #ccc;
 	margin: 0;
 }
 
 .monaco-editor {
 	padding: 0.5rem;
-}
\ No newline at end of file
+}
+
+iframe[role="wasm-runner"] {
+    opacity: 0;
```

Comment:
Hides this pixel in dark mode:
![image](https://github.com/dotnet/try/assets/38262647/ef7c7c04-a75a-4fd7-a3c6-bc0e3848cad8)